src/molgenis/capice_resources/train_data_creator/data_parsers/clinvar.py

```diff
@@ -54,6 +54,10 @@ def _obtain_gene(self, clinvar_frame: pd.DataFrame) -> None:
                 Performed inplace.
         """
         self._process_column(clinvar_frame, TrainDataCreatorEnums.GENE.value, 'GENEINFO=')
+        # For now, deleting the SYMBOL ID as VKGL does not yet export this
+        clinvar_frame[
+            TrainDataCreatorEnums.GENE.value
+        ] = clinvar_frame[TrainDataCreatorEnums.GENE.value].str.split(':', expand=True)[0]
```

Comment:
How about reporting a bug in the capice-resources repository that it doesn't take into account gene symbol aliases etc. in ClinVar data?